packages/redux/src/recentlyViewed/actions/__tests__/fetchRecentlyViewedProducts.test.ts

```diff
@@ -12,10 +14,10 @@ jest.mock('@farfetch/blackout-client/recentlyViewed', () => {
   };
 });
 
-const mockRecentlyViewedStore = (state = {}) =>
+const mockRecentlyViewedStore = (state: StoreState = {}) =>
   mockStore(
     {
-      recentlyViewed: reducer(),
+      recentlyViewed: INITIAL_STATE,
```

Comment:
Same as https://github.com/Farfetch/blackout/pull/113/files#r783166100

Reply:
Source updated, can you check again.

Reply:
It actually should be this:

```javascript
recentlyViewed: reducer(undefined, mockAction),
```

as reducers should return the initial state when the state parameter is undefined.

Reply:
Source updated, can you check again.